pkg/datapath/linux/config/config_test.go

```diff
@@ -38,8 +39,8 @@ var (
 	dummyNodeCfg  = datapath.LocalNodeConfiguration{}
 	dummyDevCfg   = testutils.NewTestEndpoint()
 	dummyEPCfg    = testutils.NewTestEndpoint()
-	ipv4DummyAddr = []byte{192, 0, 2, 3}
-	ipv6DummyAddr = []byte{0x20, 0x01, 0xdb, 0x8, 0x0b, 0xad, 0xca, 0xfe, 0x60, 0x0d, 0xbe, 0xe2, 0x0b, 0xad, 0xca, 0xfe}
+	ipv4DummyAddr = netip.MustParseAddr("192.0.2.3")
```

Comment:
I was wondering why this wouldn't return an error. Nice utility function for tests usage!